shared/switches.py

```diff
@@ -36,11 +36,11 @@ def get_switch(self, dpid):
             a kytos.core.switch.Switch() object
             False if not found
         """
-        for switch in self._switches.values():
+        for switch in self._switches.copy().values():
```

Comment:
@italovalcy it looks good. However, looking more into what the original `get_switch` method does,  maybe we can rewrite this entire method as a single line `return self._switch.get(dpid, False)`, it's not a bottleneck considering it's only going to hold a dozens of switches in practice, but it's doing an inefficient linear scan on a dict to find a key. What do you think? 

Maybe in the future we can also introduce safer helper methods on core to iterate on a shallow copy or any other safer alternative.

Reply:
> @italovalcy it looks good. However, looking more into what the original `get_switch` method does, maybe we can rewrite this entire method as a single line `return self._switch.get(dpid, False)`, it's not a bottleneck considering it's only going to hold a dozens of switches in practice, but it's doing an inefficient linear scan on a dict to find a key. What do you think?
> 
> Maybe in the future we can also introduce safer helper methods on core to iterate on a shallow copy or any other safer alternative.

Fixed on commit 7490a71